src/main/java/com/unb/tracker/WebSecurityConfig.java

```diff
@@ -12,42 +12,40 @@
 
 @Configuration
 @EnableWebSecurity
-public class WebSecurityConfig extends WebSecurityConfigurerAdapter
-{
-	@Autowired
-	private UserDetailsService userDetailsService;
+public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
+    @Autowired
+    private UserDetailsService userDetailsService;
 
-	@Bean
-	public BCryptPasswordEncoder bCryptPasswordEncoder() {
-		return new BCryptPasswordEncoder();
-	}
+    @Bean
+    public BCryptPasswordEncoder bCryptPasswordEncoder() {
+        return new BCryptPasswordEncoder();
+    }
 
-	@Override
-	protected void configure(HttpSecurity http) throws Exception
-	{
-		//TODO: Should ensure we are only exposing static content (css & js) for pages accessible to everyone (index page)
-		http
-			.csrf()
-				.disable()
-			.authorizeRequests()
-				.antMatchers("/signup").permitAll()
-				.antMatchers("/static/**").permitAll()
-				.antMatchers("/favicon.ico").permitAll()
-				.anyRequest().authenticated()
-				.and()
-			.formLogin()
-				.loginPage("/")
-				.defaultSuccessUrl("/login/success")
-				.permitAll()
-				.and()
-			.logout()
-				.logoutUrl("/logout")
-				.logoutSuccessUrl("/")
-				.permitAll();
-	}
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        //TODO: Should ensure we are only exposing static content (css & js) for pages accessible to everyone (index page)
+        http
+                .csrf()
+                .disable()
+                .authorizeRequests()
```

Comment:
I think we should undo these format changes. Keeping the indent is important in this case., because it helps us show what is modifying what :)

Reply:
agreed! The easiest way to do this are the `formatter` annotations.

For future reference, add `// @formatter:off` at the beginning of the block and `// @formatter:on` at the end to have the formatter ignore that section of code.